WebApiCoreSeed.WebApi/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.WebApi.Middleware
+{
+    /// <summary>
+    /// Middleware to handle global authorization policies
+    /// </summary>
+    public class AuthorizationMiddleware
```

Comment:
Sure, let's keep it but try to modify the line I mentioned to something that would make sense in the context of this project, example:

```C#
var isUserBlocked= context.User.HasClaim(c => c.Type == "UserStatus" && c.Value == "Blocked");
```

I think something like that would be easier to understand than the integration  user example.